From: George Hopkins <george-hopkins@null.net>
Date: Sun, 26 Jun 2016 13:32:12 +0200
Subject: Add symlink for tools/generate_shim_headers

---
 setup_links.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup_links.py b/setup_links.py
index 4cf91f7..1e0c03d 100755
--- a/setup_links.py
+++ b/setup_links.py
@@ -65,6 +65,7 @@ DIRECTORIES = [
   'third_party/WebKit',  # TODO(kjellander): Remove, see webrtc:5629.
   'tools/clang',
   'tools/generate_library_loader',
+  'tools/generate_shim_headers',
   'tools/generate_stubs',
   'tools/gn',
   'tools/gyp',
